compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -480,8 +480,8 @@ impl<'a> Resolver<'a> {
                         "const parameters may only be used as standalone arguments, i.e. `{}`",
                         name
                     ));
-                    err.note("use feature(const_generics) and feature(const_evaluatable_checked) to enable this");
                 }
+                err.note("use #![feature(const_generics)] and #![feature(const_evaluatable_checked)] to allow generic const expressions");
```

Comment:
```suggestion
                err.help("use `#![feature(const_generics)]` and `#![feature(const_evaluatable_checked)]` to allow generic const expressions");
```
Two final nits, we use backticks "`" around the feature attributes in other messages.

I quickly searched for other recommandations which mention a feature flag and they tend to use `help` instead of `note` here.

Can you change both this call to use `help` and the call in https://github.com/rust-lang/rust/blob/238994f3b1d6b05ed5c26c1b3af03723a2421700/compiler/rustc_typeck/src/check/wfcheck.rs#L332?

After this r=me :sweat_smile: